cmd/zsysd/completion.go

```diff
@@ -11,18 +12,66 @@ import (
 func installCompletionCmd(rootCmd *cobra.Command) {
 	prog := rootCmd.Name()
 	var completionCmd = &cobra.Command{
-		Use:   "completion",
-		Short: i18n.G("Generates bash completion scripts"),
-		Long: fmt.Sprintf(i18n.G(`To load completion run
+		Use:   "completion [bash|zsh|powershell]",
+		Short: i18n.G("Generates completion scripts"),
+		Long: strings.ReplaceAll(i18n.G(`To load completions:
 
-. <(%s completion)
+Bash:
 
-To configure your bash shell to load completions for each session add to your ~/.bashrc or ~/.profile:
+  $ source <(%s completion bash)
 
-. <(%s completion)
-`), prog, prog),
+  # To load completions for each session, execute once:
```

Comment:
I think I mentined it previously. bu I don’t remember, I think the top should mention that we try to automate the shell detection and that the argument of the shell variable shouldn’t be needed.

Reply:
I'll add some additional description